test/test_extractor.py

```diff
@@ -4,8 +4,6 @@
 import shutil
 import tarfile
 import tempfile
-import unittest
```

Comment:
Looks like we still need unittest with John's new test, since that's what's failing.  not sure offhand if we will also still need unittest.mock or not.